src/user/dto/login-user.dto.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { IsNotEmpty, MinLength } from "class-validator";
+
+export class LoginUserDto {
+  @ApiProperty({
+    description: "email or account as login account.",
+    example: "account",
+  })
+  @IsNotEmpty({
+    message: "email or account as login account.",
```

Comment:
should be `account field should not be empty`

Reply:
I agree with your suggestion. If write this way, will not know the error what happen.

Reply:
Revised.
[refactor: revised the error message of IsNotEmpty in login-user.dto.ts](https://github.com/Cophr/backend/pull/10/commits/ca0d39128425697494cd8047c8f88182307c2ba2)